app/controllers/permits_controller.rb

```diff
@@ -45,5 +47,15 @@ def load_adapter
     @adapter = organisation.adapter
   end
 
+  def soap_adapter_exception(e)
+    message = ''
```

Comment:
perhaps we can use: `message = if ... else .. end`; also, can we check using `is_a?` here?